bciers/apps/administration/app/components/operations/types.ts

```diff
@@ -2,11 +2,17 @@ import { OptedInOperationFormData } from "@/registration/app/components/operatio
 import { UUID } from "crypto";
 
 export interface OperationRow {
-  id: UUID;
-  bcghg_id: string;
-  name: string;
-  operator: string;
-  type: string;
+  operation__name: string;
```

Comment:
Naming them like this is kind of a pain because it affects the front end, but we put these values directly into the `.order_by` when filtering